openshift/sonar.pod.json

```diff
@@ -36,7 +36,12 @@
             "name": "GIT_BRANCH",
             "description": "Current git branch name",
             "required": true
-        }
+        },
+        {
+			"name": "UPLOADED_DOCUMENT_DEST",
```

Comment:
Indentation appears to have gone a bit wonky here

Reply:
I will re push if I can but that is not how it shows upon my local version.